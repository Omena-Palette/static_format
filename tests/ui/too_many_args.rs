use static_format::const_format;

fn main() {
    let _out = const_format!("hello world with too many provided arguments {}", "hello", "world");
}
