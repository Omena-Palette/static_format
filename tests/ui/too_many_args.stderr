error: Wrong number of arguments, expected: 1 but found 2
 --> tests/ui/too_many_args.rs:4:16
  |
4 |     let _out = const_format!("hello world with too many provided arguments {}", "hello", "world");
  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `const_format` (in Nightly builds, run with -Z macro-backtrace for more info)
